src/Jackett.Common/Definitions/demonoid.yml

```diff
@@ -7,21 +7,177 @@
+      - {id: 109, cat: TV/Anime, desc: "Anime VHS Quality"}
+      - {id: 157, cat: TV/Anime, desc: "Anime WEBRip"}
+      # Movies
+      - {id: 151, cat: Movies/UHD, desc: "Movies 4K / UHD"}
+      - {id: 110, cat: Movies/BluRay, desc: "Movies Blu-ray"}
+      - {id: 111, cat: Movies/3D, desc: "Movies Blu-ray 3D"}
+      - {id: 152, cat: Movies/BluRay, desc: "Movies Blu-ray Rip"}
+      - {id: 112, cat: Movies, desc: "Movies Cam"}
+      - {id: 113, cat: Movies/DVD, desc: "Movies DVD Full"}
+      - {id: 114, cat: Movies/DVD, desc: "Movies DVD Rip"}
+      - {id: 115, cat: Movies,/HD desc: "Movies HD 1080p / 1080i"}
+      - {id: 116, cat: Movies/HD, desc: "Movies HD 720p"}
+      - {id: 153, cat: Movies/HD, desc: "Movies HDRip"}
+      - {id: 117, cat: Movies/HD, desc: "Movies HEVC / x265"}
+      - {id: 118, cat: Movies/SD, desc: "Movies Other Quality"}
+      - {id: 119, cat: Movies, desc: "Movies Remux"}
+      - {id: 120, cat: Movies/SD, desc: "Movies Screener VHS"}
+      - {id: 121, cat: Movies/SD, desc: "Movies VHS Rip"}
+      - {id: 154, cat: Movies/WEBDL, desc: "Movies WEBRip"}
+      # Music
+      - {id: 122, cat: Audio, desc: "Music AAC"}
+      - {id: 123, cat: Audio/Lossless, desc: "Music Lossless"}
+      - {id: 124, cat: Audio/MP3, desc: "Music MP3/128Kbps"}
+      - {id: 125, cat: Audio/MP3, desc: "Music MP3/192Kbps"}
+      - {id: 126, cat: Audio/MP3, desc: "Music MP3/256Kbps"}
+      - {id: 127, cat: Audio/MP3, desc: "Music MP3/64Kbps"}
+      - {id: 128, cat: Audio/MP3, desc: "Music MP3/Over 256Kbps"}
+      - {id: 129, cat: Audio/MP3, desc: "Music MP3/Surround"}
+      - {id: 130, cat: Audio/MP3, desc: "Music MP3/Variable"}
+      - {id: 131, cat: Audio, desc: "Music MPC"}
+      - {id: 132, cat: Audio, desc: "Music OGG"}
+      - {id: 133, cat: Audio, desc: "Music Other Quality"}
+      - {id: 134, cat: Audio, desc: "Music WMA"}
+      # Music Videos
+      - {id: 135, cat: Audio/Video, desc: "Music Videos DVD Quality"}
+      - {id: 136, cat: Audio/Video, desc: "Music Videos HEVC / x265"}
+      - {id: 137, cat: Audio/Video, desc: "Music Videos Other Quality"}
+      - {id: 138, cat: Audio/Video, desc: "Music Videos TV Quality"}
+      - {id: 139, cat: Audio/Video, desc: "Music Videos VHS Quality"}
+      # Pictures
+      - {id: 140, cat: Other, desc: "Pictures Hi-Res"}
+      - {id: 141, cat: Other, desc: "Pictures Low-Res"}
+      - {id: 142, cat: Other, desc: "Pictures Mixed Quality"}
+      # TV
+      - {id: 155, cat: TV/UHD, desc: "TV 4K / UHD"}
+      - {id: 143, cat: TV/SD, desc: "TV DVD Quality"}
+      - {id: 144, cat: TV/HD, desc: "TV HD 1080p / 1080i"}
+      - {id: 145, cat: TV/HD, desc: "TV HD 720p"}
+      - {id: 146, cat: TV/HD, desc: "TV HEVC / x265"}
+      - {id: 147, cat: TV/SD, desc: "TV Other Quality"}
+      - {id: 148, cat: TV/HD, desc: "TV TV Quality"}
+      - {id: 149, cat: TV/SD, desc: "TV VHS Quality"}
+      - {id: 156, cat: TV/WEB-DL, desc: "TV WEBRip"}
```

Comment:
Hmm. seems you have switched halfway from using subcategories to quality.
I've found the current js that populates the pulldowns
```javascript
# sub categories
var sc_drop = 
 new Array(
 new Array("filters_top","category","subcategory")
,new Array(true,"0|All","0| ")
,new Array(true,"1|Applications","1|Android")
,new Array(true,"1|Applications","2|Iphone / Ipod touch")
,new Array(true,"1|Applications","3|Linux")
,new Array(true,"1|Applications","4|Mac")
,new Array(true,"1|Applications","5|Mobile phone")
,new Array(true,"1|Applications","6|PocketPC")
,new Array(true,"1|Applications","7|Windows")
,new Array(true,"2|Audio Books","8|Action")
,new Array(true,"2|Audio Books","9|Adventure")
,new Array(true,"2|Audio Books","10|Biography")
,new Array(true,"2|Audio Books","11|Childrens")
,new Array(true,"2|Audio Books","12|Computers and Technology")
,new Array(true,"2|Audio Books","13|Contemporary")
,new Array(true,"2|Audio Books","14|Cooking")
,new Array(true,"2|Audio Books","15|Crafts and Hobbies")
,new Array(true,"2|Audio Books","16|Educational")
,new Array(true,"2|Audio Books","17|Fantasy")
,new Array(true,"2|Audio Books","18|Fiction")
,new Array(true,"2|Audio Books","19|General")
,new Array(true,"2|Audio Books","20|History")
,new Array(true,"2|Audio Books","21|Horror")
,new Array(true,"2|Audio Books","22|Humor")
,new Array(true,"2|Audio Books","23|Literary")
,new Array(true,"2|Audio Books","24|Magazine")
,new Array(true,"2|Audio Books","25|Mainstream")
,new Array(true,"2|Audio Books","26|Medicine and  Health")
,new Array(true,"2|Audio Books","27|Mystery and Suspense")
,new Array(true,"2|Audio Books","28|Newspaper")
,new Array(true,"2|Audio Books","29|Nonfiction")
,new Array(true,"2|Audio Books","30|Other")
,new Array(true,"2|Audio Books","31|Paranormal")
,new Array(true,"2|Audio Books","32|Religion")
,new Array(true,"2|Audio Books","33|Romance")
,new Array(true,"2|Audio Books","34|RPG")
,new Array(true,"2|Audio Books","35|Sci-Fi")
,new Array(true,"2|Audio Books","36|Self-help")
,new Array(true,"2|Audio Books","37|Suspense")
,new Array(true,"2|Audio Books","38|Textbook")
,new Array(true,"2|Audio Books","39|Thriller")
,new Array(true,"2|Audio Books","40|Western")
,new Array(true,"2|Audio Books","41|Young Adult")
,new Array(true,"3|Books","42|Action and Adveture")
,new Array(true,"3|Books","43|Biography")
,new Array(true,"3|Books","44|Childrens")
,new Array(true,"3|Books","45|Computers and Technology")
,new Array(true,"3|Books","46|Contemporary")
,new Array(true,"3|Books","47|Cooking")
,new Array(true,"3|Books","48|Crafts and Hobbies")
,new Array(true,"3|Books","49|Educational")
,new Array(true,"3|Books","50|Fantasy")
,new Array(true,"3|Books","51|Fiction")
,new Array(true,"3|Books","52|General")
,new Array(true,"3|Books","53|History")
,new Array(true,"3|Books","54|Horror")
,new Array(true,"3|Books","55|Humor")
,new Array(true,"3|Books","56|Literary")
,new Array(true,"3|Books","57|Magazine")
,new Array(true,"3|Books","58|Mainstream")
,new Array(true,"3|Books","59|Medicine and  Health")
,new Array(true,"3|Books","60|Mystery and Suspense")
,new Array(true,"3|Books","61|Newspaper")
,new Array(true,"3|Books","62|Nonfiction")
,new Array(true,"3|Books","63|Other")
,new Array(true,"3|Books","64|Paranormal")
,new Array(true,"3|Books","65|Religion")
,new Array(true,"3|Books","66|Romance")
,new Array(true,"3|Books","67|RPG")
,new Array(true,"3|Books","68|Sci-Fi")
,new Array(true,"3|Books","69|Self-help")
,new Array(true,"3|Books","70|Suspense")
,new Array(true,"3|Books","71|Textbook")
,new Array(true,"3|Books","72|Thriller")
,new Array(true,"3|Books","73|Western")
,new Array(true,"3|Books","74|Young Adult")
,new Array(true,"4|Comics","75|Action / Adventure")
,new Array(true,"4|Comics","76|Crime")
,new Array(true,"4|Comics","77|Drama")
,new Array(true,"4|Comics","78|Fantasy")
,new Array(true,"4|Comics","79|Historical fiction")
,new Array(true,"4|Comics","80|Horror")
,new Array(true,"4|Comics","81|Illustrated novel")
,new Array(true,"4|Comics","82|Manga")
,new Array(true,"4|Comics","83|Other")
,new Array(true,"4|Comics","84|Real-Life")
,new Array(true,"4|Comics","85|Sci-Fi")
,new Array(true,"4|Comics","86|Super Hero")
,new Array(true,"5|Games","87|DOS")
,new Array(true,"5|Games","88|Dreamcast")
,new Array(true,"5|Games","89|Emulators")
,new Array(true,"5|Games","90|GameBoy")
,new Array(true,"5|Games","91|GameCube")
,new Array(true,"5|Games","92|Linux")
,new Array(true,"5|Games","93|Mac")
,new Array(true,"5|Games","94|Mobile phone")
,new Array(true,"5|Games","95|Nintendo DS")
,new Array(true,"5|Games","96|Palm")
,new Array(true,"5|Games","97|Playstation")
,new Array(true,"5|Games","98|PocketPC")
,new Array(true,"5|Games","99|PSP")
,new Array(true,"5|Games","100|Wii / Wii U")
,new Array(true,"5|Games","101|Windows")
,new Array(true,"5|Games","102|XBox")
,new Array(true,"6|Japanese Anime","103|Action")
,new Array(true,"6|Japanese Anime","104|Adventure")
,new Array(true,"6|Japanese Anime","105|Comedy")
,new Array(true,"6|Japanese Anime","106|Drama")
,new Array(true,"6|Japanese Anime","107|Fantasy")
,new Array(true,"6|Japanese Anime","108|Horror")
,new Array(true,"6|Japanese Anime","109|Other")
,new Array(true,"6|Japanese Anime","157|Romance")
,new Array(true,"6|Japanese Anime","158|Sci-Fi")
,new Array(true,"7|Miscellaneous","|")
,new Array(true,"8|Movies","151|Action")
,new Array(true,"8|Movies","110|Adventure")
,new Array(true,"8|Movies","236|Animation")
,new Array(true,"8|Movies","111|Biography")
,new Array(true,"8|Movies","152|Comedy")
,new Array(true,"8|Movies","112|Concerts")
,new Array(true,"8|Movies","113|Crime")
,new Array(true,"8|Movies","114|Documentary")
,new Array(true,"8|Movies","115|Drama")
,new Array(true,"8|Movies","116|Family")
,new Array(true,"8|Movies","153|Fantasy")
,new Array(true,"8|Movies","117|Horror")
,new Array(true,"8|Movies","118|Musical")
,new Array(true,"8|Movies","119|Mystery")
,new Array(true,"8|Movies","120|Other")
,new Array(true,"8|Movies","121|Romance")
,new Array(true,"8|Movies","154|Sci-Fi")
,new Array(true,"8|Movies","159|Short film")
,new Array(true,"8|Movies","160|Sports")
,new Array(true,"8|Movies","161|Thriller")
,new Array(true,"8|Movies","162|Trailers")
,new Array(true,"8|Movies","163|War")
,new Array(true,"8|Movies","164|Western")
,new Array(true,"9|Music","122|Alternative")
,new Array(true,"9|Music","123|Bluegrass")
,new Array(true,"9|Music","124|Blues")
,new Array(true,"9|Music","125|Childrens")
,new Array(true,"9|Music","126|Christian")
,new Array(true,"9|Music","127|Classical")
,new Array(true,"9|Music","128|Comedy")
,new Array(true,"9|Music","129|Contemporary African")
,new Array(true,"9|Music","130|Country")
,new Array(true,"9|Music","131|Dance / Disco")
,new Array(true,"9|Music","132|Drum and Bass")
,new Array(true,"9|Music","133|Electro / Techno")
,new Array(true,"9|Music","134|Folk")
,new Array(true,"9|Music","165|Gospel")
,new Array(true,"9|Music","166|Grunge")
,new Array(true,"9|Music","167|Hip-Hop / Rap")
,new Array(true,"9|Music","168|Indie")
,new Array(true,"9|Music","169|Industrial")
,new Array(true,"9|Music","170|J-Pop")
,new Array(true,"9|Music","171|Jazz")
,new Array(true,"9|Music","172|Latin American")
,new Array(true,"9|Music","173|Melodic")
,new Array(true,"9|Music","174|Metal")
,new Array(true,"9|Music","175|Other")
,new Array(true,"9|Music","176|Pop")
,new Array(true,"9|Music","177|Punk")
,new Array(true,"9|Music","178|Radio Show")
,new Array(true,"9|Music","179|Reggae")
,new Array(true,"9|Music","180|Rhythm and blues")
,new Array(true,"9|Music","181|Rock")
,new Array(true,"9|Music","182|Soul")
,new Array(true,"9|Music","183|Soundtrack")
,new Array(true,"9|Music","184|Trance")
,new Array(true,"9|Music","185|Trip Hop")
,new Array(true,"10|Music Videos","135|Alternative")
,new Array(true,"10|Music Videos","136|Bluegrass")
,new Array(true,"10|Music Videos","137|Blues")
,new Array(true,"10|Music Videos","138|Childrens")
,new Array(true,"10|Music Videos","139|Christian")
,new Array(true,"10|Music Videos","186|Classical")
,new Array(true,"10|Music Videos","187|Comedy")
,new Array(true,"10|Music Videos","188|Contemporary African")
,new Array(true,"10|Music Videos","189|Country")
,new Array(true,"10|Music Videos","190|Dance / Disco")
,new Array(true,"10|Music Videos","191|Drum and Bass")
,new Array(true,"10|Music Videos","192|Electro / Techno")
,new Array(true,"10|Music Videos","193|Folk")
,new Array(true,"10|Music Videos","194|Gospel")
,new Array(true,"10|Music Videos","195|Grunge")
,new Array(true,"10|Music Videos","196|Hip-Hop / Rap")
,new Array(true,"10|Music Videos","197|Indie")
,new Array(true,"10|Music Videos","198|Industrial")
,new Array(true,"10|Music Videos","199|J-Pop")
,new Array(true,"10|Music Videos","200|Jazz")
,new Array(true,"10|Music Videos","201|Latin American")
,new Array(true,"10|Music Videos","202|Melodic")
,new Array(true,"10|Music Videos","203|Metal")
,new Array(true,"10|Music Videos","204|Other")
,new Array(true,"10|Music Videos","205|Pop")
,new Array(true,"10|Music Videos","206|Punk")
,new Array(true,"10|Music Videos","207|Radio Show")
,new Array(true,"10|Music Videos","208|Reggae")
,new Array(true,"10|Music Videos","209|Rhythm and blues")
,new Array(true,"10|Music Videos","210|Rock")
,new Array(true,"10|Music Videos","211|Soul")
,new Array(true,"10|Music Videos","212|Soundtrack")
,new Array(true,"10|Music Videos","213|Trance")
,new Array(true,"10|Music Videos","214|Trip Hop")
,new Array(true,"11|Pictures","140|Art")
,new Array(true,"11|Pictures","141|Commercial")
,new Array(true,"11|Pictures","142|Glamour")
,new Array(true,"11|Pictures","215|Other")
,new Array(true,"11|Pictures","216|Photojournalism")
,new Array(true,"11|Pictures","217|Snapshots")
,new Array(true,"11|Pictures","218|Sports")
,new Array(true,"11|Pictures","219|Wildlife")
,new Array(true,"12|TV","155|Action")
,new Array(true,"12|TV","143|Adventure")
,new Array(true,"12|TV","144|Biography")
,new Array(true,"12|TV","145|Cartoons")
,new Array(true,"12|TV","146|Comedy")
,new Array(true,"12|TV","147|Concerts")
,new Array(true,"12|TV","148|Crime")
,new Array(true,"12|TV","149|Documentary")
,new Array(true,"12|TV","156|Drama")
,new Array(true,"12|TV","220|Family")
,new Array(true,"12|TV","221|Fantasy")
,new Array(true,"12|TV","222|Horror")
,new Array(true,"12|TV","223|Musical")
,new Array(true,"12|TV","224|Mystery")
,new Array(true,"12|TV","225|Other")
,new Array(true,"12|TV","226|Reality")
,new Array(true,"12|TV","227|Romance")
,new Array(true,"12|TV","228|Sci-Fi")
,new Array(true,"12|TV","229|Short film")
,new Array(true,"12|TV","230|Sports")
,new Array(true,"12|TV","231|Talk show")
,new Array(true,"12|TV","232|Thriller")
,new Array(true,"12|TV","233|Trailers")
,new Array(true,"12|TV","234|War")
,new Array(true,"12|TV","235|Western"));

# quality
var q_drop = 
 new Array(
 new Array("filters_top","category","quality")
,new Array(true,"0|All","0| ")
,new Array(true,"2|Audio Books","1|AAC")
,new Array(true,"2|Audio Books","2|MP3/128Kbps")
,new Array(true,"2|Audio Books","3|MP3/192Kbps")
,new Array(true,"2|Audio Books","4|MP3/256Kbps")
,new Array(true,"2|Audio Books","5|MP3/64Kbps")
,new Array(true,"2|Audio Books","6|MP3/Over 256Kbps")
,new Array(true,"2|Audio Books","7|MP3/Variable")
,new Array(true,"2|Audio Books","8|MPC")
,new Array(true,"2|Audio Books","9|OGG")
,new Array(true,"2|Audio Books","10|Other quality")
,new Array(true,"5|Games","12|Full Game")
,new Array(true,"5|Games","13|Game Rip")
,new Array(true,"6|Japanese Anime","14|4K / UHD")
,new Array(true,"6|Japanese Anime","15|DVD Quality")
,new Array(true,"6|Japanese Anime","16|HD 1080p / 1080i")
,new Array(true,"6|Japanese Anime","17|HD 720p")
,new Array(true,"6|Japanese Anime","18|HEVC / x265")
,new Array(true,"6|Japanese Anime","19|Other Quality")
,new Array(true,"6|Japanese Anime","20|VHS Quality")
,new Array(true,"6|Japanese Anime","21|WEB")
,new Array(true,"8|Movies","66|4K / UHD")
,new Array(true,"8|Movies","22|Blu-ray")
,new Array(true,"8|Movies","23|Blu-ray 3D")
,new Array(true,"8|Movies","24|Blu-ray Rip")
,new Array(true,"8|Movies","25|Cam")
,new Array(true,"8|Movies","26|DVD Full")
,new Array(true,"8|Movies","27|DVD Rip")
,new Array(true,"8|Movies","28|HD 1080p / 1080i")
,new Array(true,"8|Movies","29|HD 720p")
,new Array(true,"8|Movies","30|HDRip")
,new Array(true,"8|Movies","31|HEVC / x265")
,new Array(true,"8|Movies","32|Other Quality")
,new Array(true,"8|Movies","33|Remux")
,new Array(true,"8|Movies","34|Screener VHS")
,new Array(true,"8|Movies","35|VHS Rip")
,new Array(true,"8|Movies","36|WEB")
,new Array(true,"9|Music","67|AAC")
,new Array(true,"9|Music","37|Lossless")
,new Array(true,"9|Music","38|MP3/128Kbps")
,new Array(true,"9|Music","39|MP3/192Kbps")
,new Array(true,"9|Music","40|MP3/256Kbps")
,new Array(true,"9|Music","41|MP3/64Kbps")
,new Array(true,"9|Music","42|MP3/Over 256Kbps")
,new Array(true,"9|Music","43|MP3/Surround")
,new Array(true,"9|Music","44|MP3/Variable")
,new Array(true,"9|Music","45|MPC")
,new Array(true,"9|Music","46|OGG")
,new Array(true,"9|Music","47|Other Quality")
,new Array(true,"9|Music","48|WMA")
,new Array(true,"10|Music Videos","49|DVD Quality")
,new Array(true,"10|Music Videos","50|HEVC / x265")
,new Array(true,"10|Music Videos","51|Other Quality")
,new Array(true,"10|Music Videos","52|TV Quality")
,new Array(true,"10|Music Videos","53|VHS Quality")
,new Array(true,"11|Pictures","63|Hi-Res")
,new Array(true,"11|Pictures","64|Low-Res")
,new Array(true,"11|Pictures","65|Mixed Quality")
,new Array(true,"12|TV","54|4K / UHD")
,new Array(true,"12|TV","55|DVD Quality")
,new Array(true,"12|TV","56|HD 1080p / 1080i")
,new Array(true,"12|TV","57|HD 720p")
,new Array(true,"12|TV","58|HEVC / x265")
,new Array(true,"12|TV","59|Other Quality")
,new Array(true,"12|TV","60|TV Quality")
,new Array(true,"12|TV","61|VHS Quality")
,new Array(true,"12|TV","62|WEB"));
```

Reply:
Yeah because in this way stacking will work. But as you said we cant implement both subcategories and quality filters. I guess having subcategory only is enough.